bank system.sql

```diff
@@ -1 +1,402 @@
+/* database creation */
+CREATE DATABASE bank_system;
+GO
 
+USE bank_system;
+GO
+
+CREATE TABLE cities(
+    id INT PRIMARY KEY IDENTITY,
+    city_name VARCHAR(30) NOT NULL
```

Comment:
Read about naming convention in database
If you will use it in the query, you will have
cities.city_name
Better: Cities.CityName
Much better: Cities.Name